javascript/javascript1/week1/startup.js

```diff
@@ -0,0 +1,11 @@
+var firstWords = ["Creative ","Code ","Amazing ","Big ","Mind ","Start ","Crazy ","Hot ","Duo ","Enter "];
+var secondWords = ["ideas","Challengers","Business","Brains","Square","paper","Coders"," Developers","Centre","game"];
```

Comment:
use `const` not `var`

Reply:
I installed prettier but for some reason, it isnt working... will try to sort it..